src/ml_board/backend/restful_api/data_access.py

```diff
@@ -236,6 +256,38 @@ def get_experiment_config(self, grid_search_id: str, experiment_id: str, config_
         else:
             raise InvalidPathError(f"File path {requested_full_path} is not safe.")
 
+    def load_config(self, grid_search_id: str, config_name: str, file_type: str, experiment_id: str = None):
```

Comment:
no need to use an enum, it is just a string passed so I can add it as file extension. Also, the enum FileFormat cannot be used since FileFormat.YAML = YAML but files are stored with the extension yml. Ex: grid_search_config.yml. There will be a mismatch. o you want to make the changes everywhere then?

https://github.com/mlgym/mlgym/blob/e3455fcfac62ed3477e6592bef4969908bc0ca19/src/ml_gym/cmd_entrypoint/cmd.py#L117

Reply:
ah ok, then let's keep it for now